examples/eframe_simple.c

```diff
@@ -0,0 +1,106 @@
+/*
+ Copyright (C) 2020  The Blosc Developers
+ http://blosc.org
+ License: BSD 3-Clause (see LICENSE.txt)
+
+ Example program demonstrating use of the Blosc filter from C code.
+
+ To compile this program:
+
+ $ gcc eframe_simple.c -o eframe_simple -lblosc2
+
+ To run:
+
+ $ ./eframe_simple
+*/
+
+#include <stdio.h>
+#include <blosc2.h>
+
+#define KB  1024.
+#define MB  (1024*KB)
+#define GB  (1024*MB)
+
+#define CHUNKSIZE (1000 * 1000)
+#define NCHUNKS 100
+#define NTHREADS 4
+
+
+int main(void) {
+  static int32_t data[CHUNKSIZE];
+  static int32_t data_dest[CHUNKSIZE];
+  size_t isize = CHUNKSIZE * sizeof(int32_t);
+  int dsize;
+  int64_t nbytes, cbytes;
+  blosc2_cparams cparams = BLOSC2_CPARAMS_DEFAULTS;
+  blosc2_dparams dparams = BLOSC2_DPARAMS_DEFAULTS;
+  blosc2_schunk* schunk;
+  int i, nchunk;
+  blosc_timestamp_t last, current;
+  double ttotal;
+
+  printf("Blosc version info: %s (%s)\n", blosc_get_version_string(), BLOSC_VERSION_DATE);
+  /* Create a super-chunk container */
+  cparams.typesize = sizeof(int32_t);
+  cparams.clevel = 9;
+  cparams.nthreads = NTHREADS;
+  dparams.nthreads = NTHREADS;
+  blosc2_storage storage = {false,"dir1", .cparams=&cparams, .dparams=&dparams};
```

Comment:
For our examples, we should follow the convention of using  `.b2eframe` extension for the directories for eframes.

Also, beware of not leaving a space between parameters.

Reply:
Fixed in 9843e19